go.mod

```diff
@@ -2,6 +2,8 @@ module chainguard.dev/apko
 
 go 1.20
 
+replace github.com/chainguard-dev/go-apk => ../go-apk
```

Comment:
We remove this right?

Reply:
Blah accidentally pushed this while I was grabbing perf numbers.